src/orders/components/OrderTransactionRefundPage/components/OrderTransactionRefundTable/OrderTransactionRefundTableLine.tsx

```diff
@@ -0,0 +1,153 @@
+import { GridTable } from "@dashboard/components/GridTable";
+import { formatMoney } from "@dashboard/components/Money";
+import { OrderDetailsGrantRefundFragment } from "@dashboard/graphql";
+import useLocale from "@dashboard/hooks/useLocale";
+import { Box, Button, Input, Text } from "@saleor/macaw-ui-next";
+import React from "react";
+import { Control, Controller, FieldArrayWithId, UseFieldArrayUpdate } from "react-hook-form";
+import { FormattedMessage } from "react-intl";
+
+import { LineToRefund, OrderTransactionRefundPageFormData } from "../../OrderTransactionRefundPage";
+import { RefundQuantityChange, validateQty } from "../../utils";
+import { AlreadyRefundedHelper } from "./AlreadyRefundedHelper";
+import { refundTableMessages } from "./messages";
+
+interface OrderTransactionRefundTableLineProps {
+  control: Control<OrderTransactionRefundPageFormData, any>;
+  field: FieldArrayWithId<OrderTransactionRefundPageFormData, "linesToRefund", "id">;
+  index: number;
+  line: OrderDetailsGrantRefundFragment["lines"][number];
+  order: OrderDetailsGrantRefundFragment | undefined | null;
+  draftRefund?: OrderDetailsGrantRefundFragment["grantedRefunds"][0];
+  qtyToRefund: LineToRefund["quantity"];
+  maxQtyToRefund: number;
+  onEditReasonModal: React.Dispatch<React.SetStateAction<number | null>>;
+  refundFieldsUpdate: UseFieldArrayUpdate<OrderTransactionRefundPageFormData, "linesToRefund">;
+  onChange: (data: RefundQuantityChange, index: number) => void;
+}
+
+export const OrderTransactionRefundTableLine: React.FC<OrderTransactionRefundTableLineProps> = ({
+  control,
+  field,
+  index,
+  line,
+  order,
+  draftRefund,
+  qtyToRefund,
+  maxQtyToRefund,
+  onEditReasonModal,
+  refundFieldsUpdate,
+  onChange,
+}) => {
+  const locale = useLocale();
+
+  return (
+    <GridTable.Row key={field.id}>
+      <GridTable.Cell>
+        <Box display="grid" __gridTemplateColumns="min-content auto" gap={2} alignItems="start">
+          <Box width={8} height={8}>
+            <img src={line.thumbnail?.url} alt="product thumbnail" />
+          </Box>
+          <Box overflow="hidden" minWidth={0}>
+            <Text size={2} textOverflow="ellipsis" overflow="hidden" whiteSpace="nowrap">
+              {line.productName}
+            </Text>
```

Comment:
suggestion: The product name doesn't have ellipsis

Reply:
Fixed now